scripts/generate-bcrypt-hashed-password.py

```diff
@@ -16,7 +16,7 @@
 
 
 def hash_password(password, cost_factor):
-    return bcrypt.hashpw(bytes(password), bcrypt.gensalt(cost_factor)).decode('utf-8')
+    return bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt(cost_factor)).decode('utf-8')
```

Comment:
Noticed the 'decode' call at the end of this: is there any guarantee that bcrypt returns a string that can be interpreted as UTF-8? Isn't it just an arbitrary byte-string at this point? Presumably we're passing straight on the a URI-encoding function, in which case it might be better if this function did just return bytes to be fed in there, rather than a unicode string?

Reply:
bcrypt hashes contain a base64 string and `$` signs for version/cost factor, so they're safe to decode. We're printing them out right away, so I don't think we need to decode, but it can be useful if the function will end up being used anywhere else.